$OpenBSD: patch-libs_context_build_Jamfile_v2,v 1.3 2020/11/05 11:23:03 sthen Exp $

ppc32_sysv_elf has 2 instances of "<toolset>clang".
The second "clang" should be "gcc".

Index: libs/context/build/Jamfile.v2
--- libs/context/build/Jamfile.v2.orig
+++ libs/context/build/Jamfile.v2
@@ -280,7 +280,7 @@ alias asm_sources
      <address-model>32
      <architecture>power
      <binary-format>elf
-     <toolset>clang
+     <toolset>gcc
    ;
 
 alias asm_sources
